pkg/apis/controlplane/types.go

```diff
@@ -88,6 +88,8 @@ type GroupMember struct {
 	IPs []IPAddress
 	// Ports is the list NamedPort of the GroupMember.
 	Ports []NamedPort
+	// Service maintains the reference to the Service.
```

Comment:
maintains -> is

Reply:
Add comments about: it can only be in an AppliedToGroup and can only be a NodePort Service.

Reply:
Done.